sensors.h

```diff
@@ -91,6 +94,11 @@ class AnalogSensor
     int getValue();
 }; 
 
+struct sensorData_t {
+    bool doorOpen; // door open?
+    bool motion; // motion?
+};
```

Comment:
To my eye, it seems weird to have a struct name start with a small letter and have the `_t` on the end because I think of structs as classes without functions. But I don't code in C++ very often so am very likely not used to C++ conventions. Is this something that you saw elsewhere?